assets/css/v2/pre_fare/normal_header.scss

```diff
@@ -16,7 +16,7 @@
 }
 
 .normal-header-title {
-  width: 824px;
+  width: 1040px;
```

Comment:
Should this be 1000px? If the left screen width is 1080px and we're going for 40px of padding on either side of this text, and the left padding is already accounted for with `left: 40px;`, then the remaining width to allow here would be 1040 - 40 = 1000.

<img width="776" alt="Screen Shot 2022-03-11 at 2 02 33 PM" src="https://user-images.githubusercontent.com/63608771/157934797-cad417d5-8e36-4217-9807-923770201573.png">

Reply:
So you are definitely right, but unfortunately Downtown Crossing still wraps at 1000. So while 1040 was not picked for the right reason, we might need to use it or find a smaller width that fits that station name. What do you think?

Reply:
Hmm, there's some logic in the base header widget that's supposed to use a smaller font size if the large size wraps. If I manually change the classname to `...--small`, DTX fits. Trying to see why it's not working as expected right now.

Reply:
Actually I'll try not to get into the weeds on this and instead create a bugfix task!
With that said, I think it's best to go with the padding we want (40px on either side, which allows for a width of 1000px) and just make sure the logic to fit text within that space works.